api/CustomStateSet.json

```
@@ -28,7 +28,14 @@
           "opera": "mirror",
           "opera_android": "mirror",
           "safari": {
-            "version_added": false
+            "version_added": "preview",
+            "flags": [
+              {
+                "name": "CustomStateSet",
+                "value_to_set": "true",
+                "type": "runtime_flag"
```

Reading the BCD docs, I guess this should rather be "preference"? https://github.com/mdn/browser-compat-data/blob/main/schemas/compat-data-schema.md#flags

```suggestion
                "type": "preference"
```